packages/cojson/src/coValues/coMap.ts

```diff
@@ -91,13 +155,17 @@ export class RawCoMapView<
 
   /** @category 4. Time travel */
   atTime(time: number): this {
-    const clone = Object.create(this) as this;
-    clone.id = this.id;
-    clone.type = this.type;
-    clone.core = this.core;
-    clone.ops = this.ops;
-    clone.atTimeFilter = time;
-    return clone;
+    if (time >= this.latestTxMadeAt) {
+      return this;
```

Comment:
Optimisation for atTime where atTime is larger than all transactions in this CoMap. This especially helps with Groups where we often call atTime when determining validity of transactions in owned CoValues